console-backend-data-manager/routes/pam_login.js

```diff
@@ -39,6 +40,33 @@ module.exports = function(express, logger){
     }
   };
 
+  passport.serializeUser(function(user, done) {
+    done(null, user);
+  });
+
+  passport.deserializeUser(function(user, done) {
+    done(null, user);
+  });
+
+  passport.use('strategy-pam', new CustomStrategy(
+    function(request, callback) {
+      var username = request.body.username;
+      var pass = Base64.decode(request.body.password);
+      logger.info('pam_login username: ' + username);
+      logger.info('pam_login pass: ' + request.body.password);
```

Comment:
Should not be logging all user's passwords.

Reply:
yes right, this was for debugging, will change it. not an issue as this is encoded but should not be there anyway. thx for spotting it